libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -336,6 +337,37 @@ export class DrivingLicenseService {
     }
   }
 
+  async getQualitySignatureUri(
+    nationalId: User['nationalId'],
+  ): Promise<string | null> {
+    const image = await this.drivingLicenseApi.getQualitySignature({
+      nationalId,
+    })
+    const qualitySignature =
+      image?.data && image?.data.length > 0
```

Comment:
This is ok, just inserting fun fact :D

This could be `image?.data?.length` since `Boolean(0) →false` and `Boolean(x) →true` for any x not 0.